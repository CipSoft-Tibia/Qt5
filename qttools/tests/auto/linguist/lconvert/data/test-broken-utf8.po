# no comment
msgid ""
msgstr ""

msgid "this works"
msgstr "das geht: Ã¤"

msgid "this is broken"
msgstr "das ist kaputt: Ãi"
