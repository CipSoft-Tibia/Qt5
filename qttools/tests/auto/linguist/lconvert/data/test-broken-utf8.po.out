# no comment
msgid ""
msgstr ""
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid "this works"
msgstr "das geht: ä"

msgid "this is broken"
msgstr "das ist kaputt: �i"
